lib/protocol/sequences/Sequence.js

```diff
@@ -52,6 +52,7 @@ Sequence.prototype._packetToError = function(packet) {
   err.code = code;
   err.errno = packet.errno;
   err.sqlState = packet.sqlState;
+  err.sql = this.sql;
```

Comment:
I think @ZJONSSON has a good point on the enumerability, if you want to change it.